test/JDBC/expected/latest__preparation__linked_servers-vu-prepare.out

```diff
@@ -0,0 +1,120 @@
+
```

Comment:
Have we validated test results against SQL server? also do we have coverage for upper/mixed/lower case strings for various parameters

Reply:
Yes I have verified the results by comparing output of queries like: `SELECT * FROM t1` vs `SELECT * FROM OPENQUERY(remote_server, 'SELECT * FROM t1')`